---
  lightcmd_topic: /lightcmd
  objDetected_sub_topic: /activeObjects
  lightarray_frame: light_array
  origin_frame: base_link
  num_lights: 10
  timer_rate: 5
  y_tolerance : 0.15
  
